SSO OIDC caveat (#1737)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/custom-url-domain/overview/index.md

* If your organization has configured any SAML or WS-Fed integrated applications, review the SAML or WS-Fed SSO set-up instructions. If you want your customers to see the new custom domain rather than the Okta org domain, update those SAML or WS-Fed Service Provider integrations to use the new custom URL in the metadata.

> * If you sign a user in with your new custom URL and they try to SSO into previous OIDC integrations made with the org URL, they will be prompted to sign in again. In order to avoid this, you would need to change the issuer in these integrations to your custom URL. Both in the Okta dashboard and in your codebase.

Reviewer: ```suggestion
* If you sign a user in with your new custom URL and they try to SSO into previous OIDC integrations made with the org URL, the user is prompted to sign in again. To avoid this, change the issuer in the OIDC integrations to your custom URL in both the Okta dashboard and in your codebase.
```
